### DISCUSSION TITLE
Datahub Quantity Value not supported yet

### DISCUSSION THREAD

**Discussion by nicolaipischel** [pimcore/pimcore#8165] [Q&A / Support]
Hello we are currently using pimcore in the community version and providing access to the data through the data hubs GraphQL interface.

But we now faced a problem were we are not able to use the Quantity Value type in the datahub mutation because it seems to not be supported yet.

Is there any possibility this will be supported in the near future?

How are other users working around this problem?

**Comment by Footballschlumpf** [comment]
Hi, we have the same problem. The most of our attribute have to be "QuantityValue". Because of the large amount of datas it's importent to be able to import them. We are still searching for a workaround for Pimcore X.
